widgets/projection/powder/powder.py

```diff
@@ -1,8 +1,44 @@
 from powder_ui import Ui_Form
 from PyQt4.QtGui import QWidget
+from presenters.powder_projection_presenter import PowderProjectionPresenter
+from models.projection.powder.mantid_projection_calculator import MantidProjectionCalculator
+from views.powder_projection_view import PowderView
+from command import Command
 
 
-class PowderWidget(QWidget,Ui_Form):
+class PowderWidget(QWidget,Ui_Form,PowderView):
+    """This widget is not usable without a main window which implements mainview"""
```

Comment:
No problem doing that. 
However the widget is not in a usable state (no presenter) until someone calls set_main_window with something that implements main view